lib/rspec/core/hooks.rb

```diff
@@ -265,7 +266,8 @@ def prepend_before(*args, &block)
       # This is the reverse of the order in which `before` hooks are run.
       # Similarly, if more than one `after` is declared within any one scope,
       # they are run in reverse order of that in which they are declared. Also
-      # `around` hooks will all have run before any after hooks are invoked.
+      # `around` hooks will run after any `after` example hooks are invoked but
+      # before any `after` context hooks.
```

Comment:
Just in case you will find my previous suggestion conviencing enough, here's a similar change to this paragraph just to keep the wording more or less consistent:
```diff
      # Similarly, if more than one `after` is declared within any one scope,
      # they are run in reverse order of that in which they are declared. Also
+      # `around` hooks will run after any `after` hooks with `:example` scope
+      # are invoked but before any `after` hooks with `:context` scope.
-      # `around` hooks will run after any `after` example hooks are invoked but
-      # before any `after` context hooks.
```